src/Integration/Binding/CSharpVBBindingOperation.Writer.cs

```diff
@@ -70,9 +70,13 @@ internal partial class CSharpVBBindingOperation
                 return existingRuleSetPath;
             }
 
+            if (ShouldIgnoreConfigureRuleSetValue(currentRuleSetPath))
```

Comment:
Not changed in this commit, but the scope of `ShouldIgnoreConfigureRuleSetValue` doesn't need to be public.